examples/sri-1/components/workgroup/invitation/invitation.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Agreement } from "../storage/agreement";
+import { BpiSubject } from "../identity/bpiSubject";
+export interface IInvitation {
+    id: string;
```

Comment:
Add a TODO in places where we know we need to make changes. id will be UUID as soon as #491 is done and we have proper package management.

Reply:
TODOs added from id to uuid change at #491